### PR TITLE
[FAB-18109] Update peer chaincode invoke

### PR DESCRIPTION
**Pull request by pamandrejko**
Need to describe how to target a specific smart contract inside
a chaincode package.
- Added a new example to peer chaincode invoke
- Added instructions to writing your first application tutorial.

Content is already included in the Developing Apps documentation.

Signed-off-by: pama-ibm <pama@ibm.com>

#### Type of change

- Documentation update

#### Description

User feedback that they could not find instructions for how to use the peer chaincode invoke
command when chaincode package includes multiple smart contracts.

- Added a new example to peer chaincode invoke
- Added instructions to writing your first application tutorial.

(Content is already included in the Developing Apps documentation.)

### CODE REVIEW

**Review by denali49 — DISMISSED** [review]
LGTM, thanks Pam.

File: docs/source/write_first_app.rst [hunk 1]
```diff
@@ -340,9 +340,16 @@ to the Contract using the contract name and channel name via Gateway:
     // Build a network instance based on the channel where the smart contract is deployed
     const network = await gateway.getNetwork(channelName);
 
+
     // Get the contract from the network.
     const contract = network.getContract(chaincodeName);
 
+When a chaincode package includes multiple smart contracts, on the `getContract() API <https://hyperledger.github.io/fabric-sdk-node/release-2.2/module-fabric-network.Network.html#getContract>`__ you can specify the both the name of the chaincode package and a specific smart contract to target. For example:
```
Comment (denyeart) [inline docs/source/write_first_app.rst]:
```suggestion
When a chaincode package includes multiple smart contracts, on the `getContract() API <https://hyperledger.github.io/fabric-sdk-node/release-2.2/module-fabric-network.Network.html#getContract>`__ you can specify both the name of the chaincode package and a specific smart contract to target. For example:
```

File: docs/wrappers/peer_chaincode_postscript.md [hunk 2]
```diff
@@ -66,6 +66,18 @@ Here is an example of the `peer chaincode invoke` command:
...
+Here is an example of how to format the `peer chaincode invoke` command when the chaincode package includes multiple smart contracts.
+
+  * If you are using the [contract-api](https://www.npmjs.com/package/fabric-contract-api), the name you pass to `super("MyContract")` can be used as a prefix.
+
+  ```
+  peer chaincode invoke -C $CHANNEL_NAME -n $CHAINCODE_NAME -c '{ "Args": ["MyContract:methodName", "{}"] }'
+
+  peer chaincode invoke -C $CHANNEL_NAME -n $CHAINCODE_NAME -c '{ "Args": ["MyOtherContract:methodName", "{}"] }'
+
+  ```
```
Comment (denyeart) [inline docs/wrappers/peer_chaincode_postscript.md]: Should be indented if you want to follow the precedent of the other examples.

**Review by denyeart — COMMENTED** [review]
Thanks, looks good, just two nits...

File: docs/wrappers/peer_chaincode_postscript.md (hunk 2 above)
Comment (pamandrejko) [reply docs/wrappers/peer_chaincode_postscript.md]: Done.

**Review by denali49 — APPROVED** [review]
LGTM, thanks Pam!

**Review by denyeart — APPROVED** [review]

**Comment by pamandrejko** [comment]
@Mergifyio backport release-2.2

**Comment by pamandrejko** [comment]
@Mergifyio backport release-2.2

**Pull request merged**